### ISSUE TITLE
hallowing does not boot with build 7c443fb.

### ISSUE THREAD

**Issue by jerryneedell** [adafruit/circuitpython#1514]
I tried adafruit-circuitpython-hallowing_m0_express-en_US-20190131-7c443fb.uf2 and the Hallowing does not boot 

dmesg reports 
```
163403.942122] FAT-fs (sdc): unable to read boot sector to mark fs as dirty
```

adafruit-circuitpython-hallowing_m0_express-en_US-20190131-63d4561.uf2  boots fine.

**Comment by jerryneedell** [comment]
latest build also does not boot ;
https://adafruit-circuit-python.s3.amazonaws.com/bin/hallowing_m0_express/en_US/adafruit-circuitpython-hallowing_m0_express-en_US-20190201-7d039a5.uf2

same dmesg report
i

**Comment by tannewt** [comment]
I think I've fixed this in #1513

**Comment by jerryneedell** [comment]
yes you did ```
Press any key to enter the REPL. Use CTRL-D to reload.
Adafruit CircuitPython 4.0.0-beta.1-38-g845783a45 on 2019-02-01; HalloWing M0 Express with samd21g18
>>> 

```
